monitor-agent/deploy.sh

```
@@ -0,0 +1,159 @@
+        log_info "Restarting $REGION ($SERVER)..."
+        ssh $SERVER "docker restart frogstats-monitor-agent"
+        ;;
+        
+    stop)
+        log_info "Stopping $REGION ($SERVER)..."
+        ssh $SERVER "docker stop frogstats-monitor-agent"
+        ;;
+        
+    *)
+        echo "Usage: $0 [region] [action]"
+        echo "Regions: ${!REGIONS[@]}"
```

Shellcheck (0.10.0)

[error] Argument mixes string and array. Use * or separate argument.

(SC2145)